db-engine/src/main/java/com/splicemachine/db/iapi/services/property/PropertyUtil.java

```diff
@@ -92,7 +92,8 @@
 		"derby.language.logQueryPlan",
 		Property.LOCKS_ESCALATION_THRESHOLD,
 		Property.DATABASE_PROPERTIES_ONLY,
-		Property.DEFAULT_CONNECTION_MODE_PROPERTY
+		Property.DEFAULT_CONNECTION_MODE_PROPERTY,
+		"derby.language.unSatTreePruningOff"
 	};
```

Comment:
This would create another dimension to testing.  It would have to be always on for the commit to be allowed.  We are already stretched with control vs. Spark.